rootfs/etc/profile.d/aws-okta.sh

```diff
@@ -63,7 +63,12 @@ if [ "${AWS_OKTA_ENABLED}" == "true" ]; then
 
 		shift
 		if [ $# -eq 0 ]; then
+			history -a # append history to file so it is available in subshell
 			aws-okta exec ${AWS_OKTA_ARGS[@]} $role -- bash -l
+			# read history from the subshell into the parent shell
+			# history -n does not work when HISTFILESIZE > HISTSIZE
+			history -c
```

Comment:
Why clear the history first ?

Reply:
`history -r` will append the contents of the history file to the end of the history in memory. That will cause duplicates if the history is not cleared first.